tests/dotnet/UnitTests/TestBaseClass.cs

```diff
@@ -118,5 +118,22 @@ protected string GetInfoPListPath (ApplePlatform platform, string app_directory)
 			}
 		}
 
+		protected void AssertBundleAssembliesStripStatus (string appPath, bool shouldStrip)
+		{
+			var assemblies = Directory.GetFiles (appPath, "*.dll");
```

Comment:
I think this only finds files in the root directory, which is not necessarily where assemblies will be for fat apps (nor for macOS or Mac Catalyst, but those shouldn't be stripped anyway). IMHO it would be better to get all assemblies in all subdirectories.

Reply:
Done